src/modules/UI/scenes/SendConfirmation/action.js

```diff
@@ -109,7 +109,16 @@ export const updateMaxSpend = () => (dispatch: Dispatch, getState: GetState) =>
   const spendInfo = getSpendInfo(state)
 
   getMaxSpendable(edgeWallet, spendInfo)
-    .then(nativeAmount => dispatch(createTX({ nativeAmount }, true)))
+    .then(nativeAmount => {
+      const state = getState()
+      const spendInfo = getSpendInfo(state, { nativeAmount })
+      const authRequired = getAuthRequired(state, spendInfo)
+
+      dispatch(reset())
```

Comment:
Are you able to combine these three dispatches into one? Or two?

Reply:
im not sure how to do that
reset enables the scene to detect the spendInfo/transaction as "new" and not "updated", adn this triggers the TextInput focusing, gross, i know
if you can come up with a better way, im all ears

until then, 3 dispatches instead of 1 for an action that rarely ever happens, and would only happen once, i think is okay for now